website/docs/r/alert_channel_victor_ops.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_victor_ops"
+description: |-
+  Create and manage Victor Ops Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_victor_ops
```

Comment:
ugh... I hate to do this but I just found out that VictorOps have a Terraform provider and they called it `victorops` and not `victor_ops` so...... 
```suggestion
# lacework\_alert\_channel\_victorops
```